.github/workflows/centraldb_docker_publish.yaml

```
@@ -9,13 +9,13 @@ on:
       - releasing/version/VERSION
 
 env:
-  DOCKER_USER: kubeflownotebookswg
-  IMG: kubeflownotebookswg/centraldashboard
+  DOCKER_USER: kubeflow
+  IMG: ghcr.io/kubeflow/kubeflow/central-dashboard
```

Can we use the updated repository according to our discussion here: https://github.com/kubeflow/kubeflow/pull/7706/files#r1992231043 ?
```suggestion
  IMG: ghcr.io/kubeflow/dasboard/central-dashboard
```

---

We will use that repo once we migrate to `kubeflow/dashboard`, the `ghcr.io/kubeflow/kubeflow/central-dashboard` already exists.